core-bundle/tests/Twig/Extension/ContaoExtensionTest.php

```diff
@@ -342,6 +342,50 @@ public function provideTemplateNames(): \Generator
         yield 'installation-bundle template' => ['@ContaoInstallation/database.html.twig'];
     }
 
+    /**
```

Comment:
We need to delegate to functions that are marked `@internal` in Twig. This tests makes sure that nothing changes in those internal implementations without being unnoticed by us. We can get rid of a lot of this when we're dropping input encoding in the future.